site/en/docs/devtools/wasm/index.md

```diff
@@ -0,0 +1,117 @@
+1. To set a [line-of-code breakpoint](/docs/devtools/javascript/breakpoints/#loc), click a line number in the column to the left of the **Editor**, for example, on line 38.
+
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/1Bsy2zi8gmeo3XJX4X3a.png", alt="A line-of-code breakpoint set on line 38.", width="800", height="490" %}
+
+1. Run the code again. The execution pauses before the line with the breakpoint.
+
+While paused, try the following:
+
+- In **Sources** > **Editor**, hover over a variable to see its value in a tooltip.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/RlXe0bF3r7DAb1gOnVxJ.png", alt="The value of a variable in a tooltip.", width="800", height="490" %}
+
+- In **Sources** > **Call Stack**, view function names as they are in the source.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ThxH46EBPaLxqhNAOzXU.png", alt="The main function in the Call Stack.", width="800", height="428" %}
+
+- In **Sources** > **Scope**, view local and global variables, their types, and values.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/7kAzmM4kHVKnwyK9XbeM.png", alt="The Scope pane with local variables and their values.", width="800", height="442" %}
+
+- In **Console**, output variables and objects that are hard to navigate to in **Scope**:
+
+  - Deeply nested variables, for example, indexed items in big arrays.
+  - Complex objects, including those you can access with pointers (`->`). Expand them to inspect.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ieeESEUXLsEsP7jl7evq.png", alt="A nested variable and a complex object expanded in the Console.", width="800", height="668" %}
+
+- In **Sources** > **Scope**, click the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/jgJz0iMIw4XhDqhdwYP2.svg", alt="Memory.", width="20", height="20" %} icon to open the **Memory Inspector** and inspect the raw bytes of object memory. For more information, see [WebAssembly memory inspection](/docs/devtools/memory-inspector/#wasm).
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/mD5y0xEJJ7kCfclIGwWS.png", alt="Inspecting the memory of a variable.", width="800", height="633" %}
+
+## Profile performance {: #perf }
+
+With DevTools set up and open, the code Chrome runs isn't optimized. It's [tiered down](https://v8.dev/docs/wasm-compilation-pipeline#debugging) to give you better debugging experience.
+
+In this case, you can't rely on `console.time()` and `performance.now()`  in your code to profile performance. Instead, use the [**Performance**](/docs/devtools/performance/reference/#record) panel to profile.
+
+Alternatively, you can run your profiling code without opening DevTools, then open it to inspect the messages in the **Console**.
+
+## Separate the debug information {: #separate-debug }
+
+To speed up compilation and loading times but still have a better debugging experience, you can split out the debug information into a separate `.wasm` file. You can either keep this file locally or host it on a separate server.
```

Comment:
-gseparate-dwarf arguably only affects loading time, not compilation. Would it make sense to link to [Debugging WebAssembly Faster] here, or is it enough that that's referenced at the bottom?